WordPress/Classes/ViewRelated/Reader/ReaderSavedPostsViewController.swift

```diff
@@ -50,6 +57,11 @@ final class ReaderSavedPostsViewController: UITableViewController {
         refreshNoResultsView()
     }
 
+    override func viewWillDisappear(_ animated: Bool) {
+        super.viewWillAppear(animated)
+        postCellActions?.clear()
```

Comment:
I'm not sure that `viewWillDisappear` is the right place to clear the actions. This will currently clear the actions on any navigation away from this screen – whether that's tapping on a different tab bar tab, or tapping into the detail of a saved post.

On Android, I noticed that they only remove the 'undo' cells if you navigate _back_ out of the Saved Posts list. If you tap into the detail of a post and then back to the Saved Posts list, the undo items are still there.

Would `deinit` work? Or `viewDidDisappear`, but check that we're no longer the top view controller in the navigation stack?